agenda-services/src/main/java/org/exoplatform/agenda/rest/AgendaEventRest.java

```diff
@@ -1324,11 +1324,13 @@ public Response dismissEventDateOption(
   }
 
   @Path("{eventId}/response/send")
-  @POST
+  @GET
+  // This query modify data, but to accept events, so the risk of attack via this bias is low.
+  // a hacker will not delete or modify important data via this request.
```

Comment:
```suggestion
  // This query modify data, and use GET verb. It is used in mail notification to accept events with a link, and it is not possible to make a POST by this way. So we keep GET here and assume that it will not be protected by CSRF protection
```